libpermian/events/structures/builtin.py

```diff
@@ -3,11 +3,12 @@
 
 @EventStructuresFactory.register('product')
 class ProductStructure(BaseStructure):
-    def __init__(self, settings, name, major, minor):
+    def __init__(self, settings, name, major, minor, flag=None):
```

Comment:
It seems the support for the third version number is missing. It would be great to have this fixed before introducing the "flag".